### ISSUE TITLE
++ got worse

### ISSUE THREAD

**Issue by OZ1** [ofolis/consolig#25]
I like the ++ ligature idea in general, but the current implementation looks dirty, because the vertical dashes now span accross two pixels.
![image](https://user-images.githubusercontent.com/1448985/112629335-3fd8ea00-8e45-11eb-94c6-ca91a90e1875.png)

Compare it with the perfectly fitted single plus character: 
![image](https://user-images.githubusercontent.com/1448985/112629622-99411900-8e45-11eb-9482-c1b74bc2d1fe.png)

**Comment by ofolis** [comment]
Yeah, the horizontal grid alignment could use some work. I'll have a look at this one.
